src/types/Species.ts

```diff
@@ -149,3 +151,5 @@ export enum SpeciesRequestError {
   // setting up the request. In other words, there was a developer error or server outage.
   RequestFailed = 'AN_UNRECOVERABLE_ERROR_OCCURRED',
 }
+
+export type SpeciesDetails = components[typeof schemas]['SpeciesLookupDetailsResponsePayload'];
```

Comment:
i believe `'schemas'` might work as well.. we are using `typeof schemas` in many places.. not sure why.. a lot of it was mostly copy paste lol

Reply:
Yes, i copied/pasted that